guides/introduction/overview.md

```diff
@@ -17,28 +17,28 @@ Here are a few resources that might be helpful:
 
 Absinthe's functionality generally falls into two broad areas. You can read more about the details in the guides provided as part of this documentation and in the related packages/projects:
 
-* [Defining Schemas](schemas.md). A schema:
+* [Defining Schemas](../schemas.md). A schema:
```

Comment:
Are you sure these are broken? The current links work fine in hexdocs:

https://hexdocs.pm/absinthe/overview.html

Reply:
I reverted these changes. I originally made them because I was blindly following Webstorm's suggestions, but if they're working fine already then my changes aren't needed.